core/src/main/java/discord4j/core/object/data/stored/BaseGuildBean.java

```diff
@@ -269,6 +276,8 @@ public void setVerificationLevel(final int verificationLevel) {
         this.verificationLevel = verificationLevel;
     }
 
+    public int getPremiumTier() { return premiumTier; }
```

Comment:
Needs reformatting. We use this code style for the project https://github.com/Discord4J/meta/blob/master/Discord4J.xml

Happens a couple of times below too.

Reply:
fixed in #b8949e2